mk/checks.mk

```diff
@@ -4,12 +4,14 @@
 
 
 #***********************************
-### Check podman registry login
+### Check podman registry login and podman quay login
```

Comment:
Both `registry.redhat.io` and `quay.io` are container registries so this change does not make sense to me. You can update the text by just putting the `registry` word to plural making it `registries`